src/Symfony/Component/Serializer/Exception/MissingConstructorArgumentException.php

```diff
@@ -11,31 +11,31 @@
 
 namespace Symfony\Component\Serializer\Exception;
 
-class MissingConstructorArgumentException extends MissingConstructorArgumentsException
+class MissingConstructorArgumentException extends RuntimeException
```

Comment:
can't we get rid of this class altogether and keep only `MissingConstructorArgumentsException`
also: please mind fabbot ;)